.github/workflows/ci.yml

```
@@ -370,3 +370,27 @@ jobs:
         uses: softprops/action-gh-release@v1
         with:
           files: 'artifact/*/Zoo*'
+          
+  announce_release:
+    needs: [publish-apps-release]
```

```suggestion
    needs: [publish-apps-release]
    if: ${{ github.event_name == 'release' || github.event_name == 'schedule' }}
```

@pierremtb might it be worth adding the same if statement that `publish-apps-release` has, or is teh `needs` enough to cover that?

---

needs should do the trick (if you check the last commit, you'll see that both are shown as skipped)